pandas/core/generic.py

```diff
@@ -10329,6 +10329,28 @@ def _doc_parms(cls):
 %(examples)s
 """
 
+_num_doc_mad = """
+%(desc)s
+
+Parameters
+----------
+axis : %(axis_descr)s
+    Axis for the function to be applied on.
+skipna : bool, default True
```

Comment:
Thanks @r0cketr1kky 

Isn't the default for `skipna` `None` rather than `True`?
```
def mad(self, axis=None, skipna=None, level=None):
```

Reply:
Yes, you're right. I will change this too.